shared/teams/add-members-wizard/confirm.tsx

```diff
@@ -17,10 +17,24 @@ import {ModalTitle, ChannelsWidget} from '../common'
 import {pluralize} from '../../util/string'
 import logger from '../../logger'
 
+type DisabledRoles = React.ComponentProps<typeof FloatingRolePicker>['disabledRoles']
+const disabledRolesForEmailsPhones = {
+  owner: null,
```

Comment:
Designs have this not showing for the group setting at all, not sure that really makes sense though.

Reply:
Even when you're adding multiple keybase users? I've wanted to add them as owners before though :|